src/data/constants/index.js

```diff
@@ -0,0 +1,3 @@
+export const VERIFIED_TRACK = { name: 'Verified', key: 'verified' };
+export const AUDIT_TRACK = { name: 'Audit', key: 'audit' };
+export const PROFESSIONAL_TRACK = { name: 'Professional', key: 'professional' };
```

Comment:
When are we gonna start marking strings for translation?

Reply:
That is a good question.  I will add that to the list of things to plan for.